src/nft/hooks/useBag.ts

```diff
@@ -57,6 +58,10 @@ export const useBag = create<BagState>()(
         set(({ bagExpanded }) => ({
           bagExpanded: !bagExpanded,
         })),
+      openBag: () =>
+        set(() => ({
+          bagExpanded: true,
```

Comment:
this is for when a user clicks "buy".  The only other option was toggle

Reply:
why is this necessary? you could just check if `bagExpanded` and if its not you `toggleBag`